plugin_test.go

```diff
@@ -97,6 +98,27 @@ var commits = []struct {
 		file: "README",
 		data: "Hello World!\n\nsomething is changed!\n",
 	},
+	// ### test lfs, please do not change order, otherwise TestCloneNonEmpty will fail ###
+	// chekout with lfs skip
```

Comment:
this should read "checkout"

Please make sure you fixed it before marking the issue as resolved!